app/scripts/controllers/citationInfo.ctrl.js

```diff
@@ -7,6 +7,24 @@ angular.module('yourStlCourts').controller('CitationInfoCtrl', function (faqData
   ctrl.paymentUrl = "";
   ctrl.citationCourtLocations = {};
 
+  ctrl.issueMultiplePeopleWarning = function(){
```

Comment:
So....we don't want to just show groupings of tickets by people? If we always show groupings, then it handles every case and we don't have to show a message. It's implicitly indicated separately.